lib/runc_sandbox.ml

```diff
@@ -269,6 +279,11 @@ let run ~cancelled ?stdin:stdin ~log t config results_dir =
   let json_config = Json_config.make config ~config_dir:tmp ~results_dir t in
   Os.write_file ~path:(tmp / "config.json") (Yojson.Safe.pretty_to_string json_config ^ "\n") >>= fun () ->
   Os.write_file ~path:(tmp / "hosts") "127.0.0.1 localhost builder" >>= fun () ->
+  List.fold_left
+    (fun r (key, value, _) -> r >>= fun () -> Os.write_file ~path:(tmp / "secret-" ^ key ) value)
```

Comment:
It might be better to number the secrets on the host, so that the user doesn't have control of the name. Especially for names with `/../../` in them...